test/interactions/doubleClickInteractionTests.ts

```diff
@@ -1,35 +1,56 @@
 ///<reference path="../testReference.ts" />
 
 describe("Interactions", () => {
-  describe("DoubleClick", () => {
-    let SVG_WIDTH = 400;
-    let SVG_HEIGHT = 400;
+  describe("DoubleClick Interaction", () => {
 
-    describe("onDblClick generic callback", () => {
+    describe("Basic Usage", () => {
+      let userClickPoint: Plottable.Point;
```

Comment:
Unsure of the name `userClickPoint` as the `user` prefix is not useful.  I think we should retain the previous variable name or just `point`